src/main/java/it/trade/api/StatelessTradeItApiClient.java

```diff
@@ -58,132 +58,132 @@ protected TradeItApi createTradeItApi(TradeItEnvironment environment, final Inte
 //        httpClientBuilder.networkInterceptors().add(new LoggingInterceptor()); //uncomment if you want some request/response logs
 
         Retrofit retrofit = new Retrofit.Builder()
-            .baseUrl(environment.getBaseUrl())
-            .client(httpClientBuilder.build())
-            .addConverterFactory(GsonConverterFactory.create())
-            .build();
+                .baseUrl(environment.getBaseUrl())
+                .client(httpClientBuilder.build())
+                .addConverterFactory(GsonConverterFactory.create())
+                .build();
```

Comment:
A bunch of these changes are doubling the whitespace. Can you check your IntelliJ settings and remove these changes unless it's a whitespace change we want?